AWS/README.md

```diff
@@ -38,50 +38,337 @@
+* Aws eks get all namespaces.py
+* Aws eks get all pods.py
+* Aws eks get deployments name.py
+* Aws eks get node cpu memory.py
+* Aws eks get nodes.py
+* Aws eks get not running pods.py
+* Aws eks get pod cpu memory.py
+* Aws eks get pod status.py
+* Aws eks get running pods.py
+* Aws eks run kubectl cmd.py
+* Aws emr get instances.py
 * Aws execute cli command.py
+* Aws execute command ssm.py
 * Aws filter all ec2 instances.py
 * Aws filter ec2 by tags.py
+* Aws filter ec2 by vpc.py
 * Aws filter ec2 without lifetime tag.py
 * Aws filter all manual database snapshots.py
 * Aws filter public s3 buckets.py
+* Aws filter target groups by tags.py
 * Aws filter unattached ebs volume.py
 * Aws filter unencrypted s3 buckets.py
 * Aws filter untagged ec2 instances.py
 * Aws filter unused keypairs.py
+* Aws get alarms list.py
 * Aws get alb listeners without http redirect.py
 * Aws get ec2 instances all.py
 * Aws get untagged resources.py
 * Aws get autoscaling instances.py
+* Aws get bucket size.py
+* Aws get cloudwatch ebs.py
+* Aws get cloudwatch ec2.py
+* Aws get cloudwatch ec2 cpuutil.py
+* Aws get cloudwatch metrics applicationelb.py
+* Aws get cloudwatch metrics classic elb.py
 * Aws get cloudwatch metrics dynamodb.py
+* Aws get cloudwatch metrics ec2autoscaling.py
+* Aws get cloudwatch metrics gatewayelb.py
+* Aws get cloudwatch metrics lambda.py
+* Aws get cloudwatch metrics network elb.py
+* Aws get cloudwatch metrics rds.py
+* Aws get cloudwatch metrics redshift.py
+* Aws get cloudwatch metrics sqs.py
+* Aws get cloudwatch statistics.py
+* Aws get ec2 instances all.py
+* Aws get ecs services status.py
 * Aws get handle.py
+* Aws get instance detail with private dns name.py
 * Aws get instance details.py
+* Aws get instances.py
 * Aws get internet gateway by vpc.py
-* Aws get nat gateway by vpc.py
 * Aws get publicly accessible db instances.py
 * Aws get publicly accessible db snapshots.py
 * Aws get s3 bucket list.py
 * Aws get schedule to retire instances.py
+* Aws get secret from secretmanager.py
+* Aws get security group details.py
 * Aws get sts caller identity.py
 * Aws get tags of resources.py
+* Aws get unhealthy instances.py
 * Aws launch instance from ami.py
 * Aws list access keys.py
 * Aws list application loadbalancers.py
 * Aws list attached user policies.py
+* Aws loadbalancer list instances.py
+* Aws make bucket public.py
 * Aws modify listener http redirection.py
 * Aws modify public db snapshots to private.py
+* Aws postgresql get configured max connections.py
+* Aws postgresql plot active connections.py
+* Aws put bucket cors.py
+* Aws put bucket policy.py
+* Aws read object.py
+* Aws register instances.py
 * Aws restart instances.py
 * Aws run instances.py
 * Aws stop instances.py
+* Aws target group list instances.py
+* Aws target group list unhealthy instances.py
+* Aws target group register unregister instances.py
 * Aws terminate ec2 instances.py
 * Aws update access keys.py
+* Aws upload file to s3.py
+
```

Comment:
Got it. Guess the above section can be removed from the README.md?

Reply:
fixed... one time issue as we change the code,